aws/resource_aws_glue_job_test.go

```diff
@@ -397,6 +397,7 @@ func TestAccAWSGlueJob_PythonShell(t *testing.T) {
 					resource.TestCheckResourceAttr(resourceName, "max_capacity", "0.0625"),
 					resource.TestCheckResourceAttr(resourceName, "command.#", "1"),
 					resource.TestCheckResourceAttr(resourceName, "command.0.script_location", "testscriptlocation"),
+					resource.TestCheckResourceAttr(resourceName, "command.0.python_version", "3"),
```

Comment:
When no python_version is specified the default response from the API will be version 2. Once support is added for glue version 1.0 the default value will be 3. 

```suggestion
					resource.TestCheckResourceAttr(resourceName, "command.0.python_version", "2"),
```

Reply:
Let's update this test to account for the three PythonVersion use cases (i.e Not Set, Set to 2, Set to 3). 

Start by adding the following test configuration under the existing ` testAccAWSGlueJobConfig_PythonShell` function
```go                                                                                                                                                                                                                                                                     
func testAccAWSGlueJobConfig_PythonShellWithVersion(rName, pythonVersion string) string {                                                                                                                                                                                                            
    return fmt.Sprintf(`                                                                                                                                                                                                                                                                              
  %s                                                                                                                                                                                                                                                                                                  
                                                                                                                                                                                                                                                                                                      
  resource "aws_glue_job" "test" {                                                                                                                                                                                                                                                                    
    name         = %q                                                                                                                                                                                                                                                                                 
    role_arn     = "${aws_iam_role.test.arn}"                                                                                                                                                                                                                                                         
    max_capacity = 0.0625                                                                                                                                                                                                                                                                             
                                                                                                                                                                                                                                                                                                      
    command {                                                                                                                                                                                                                                                                                         
      name            = "pythonshell"                                                                                                                                                                                                                                                                 
      script_location = "testscriptlocation"                                                                                                                                                                                                                                                          
      python_version =  %q                                                                                                                                                                                                                                                                            
    }                                                                                                                                                                                                                                                                                                 
                                                                                                                                                                                                                                                                                                      
    depends_on = ["aws_iam_role_policy_attachment.test"]                                                                                                                                                                                                                                              
  }                                                                                                                                                                                                                                                                                                   
  `, testAccAWSGlueJobConfig_Base(rName), rName, pythonVersion)                                                                                                                                                                                                                                       
  }
```


Then update the test to look like the following: 

```go
  func TestAccAWSGlueJob_PythonShell(t *testing.T) {                                                                                                                                                                                                                                                  
    var job glue.Job                                                                                                                                                                                                                                                                                  
                                                                                                                                                                                                                                                                                                      
    rName := fmt.Sprintf("tf-acc-test-%s", acctest.RandString(5))                                                                                                                                                                                                                                     
    resourceName := "aws_glue_job.test"                                                                                                                                                                                                                                                               
                                                                                                                                                                                                                                                                                                      
    resource.ParallelTest(t, resource.TestCase{                                                                                                                                                                                                                                                       
      PreCheck:     func() { testAccPreCheck(t) },                                                                                                                                                                                                                                                    
      Providers:    testAccProviders,                                                                                                                                                                                                                                                                 
      CheckDestroy: testAccCheckAWSGlueJobDestroy,                                                                                                                                                                                                                                                    
      Steps: []resource.TestStep{                                                                                                                                                                                                                                                                     
        {                                                                                                                                                                                                                                                                                             
          Config: testAccAWSGlueJobConfig_PythonShell(rName),                                                                                                                                                                                                                                         
          Check: resource.ComposeTestCheckFunc(                                                                                                                                                                                                                                                       
            testAccCheckAWSGlueJobExists(resourceName, &job),                                                                                                                                                                                                                                         
            resource.TestCheckResourceAttr(resourceName, "max_capacity", "0.0625"),                                                                                                                                                                                                                   
            resource.TestCheckResourceAttr(resourceName, "command.#", "1"),                                                                                                                                                                                                                           
            resource.TestCheckResourceAttr(resourceName, "command.0.script_location", "testscriptlocation"),                                                                                                                                                                                          
            resource.TestCheckResourceAttr(resourceName, "command.0.python_version", "2"),                                                                                                                                                                                                            
            resource.TestCheckResourceAttr(resourceName, "command.0.name", "pythonshell"),                                                                                                                                                                                                            
          ),                                                                                                                                                                                                                                                                                          
        },                                                                                                                                                                                                                                                                                            
        {                                                                                                                                                                                                                                                                                             
          ResourceName:      resourceName,                                                                                                                                                                                                                                                            
          ImportState:       true,                                                                                                                                                                                                                                                                    
          ImportStateVerify: true,                                                                                                                                                                                                                                                                    
        },                                                                                                                                                                                                                                                                                            
        {                                                                                                                                                                                                                                                                                             
          Config: testAccAWSGlueJobConfig_PythonShellWithVersion(rName, "2"),                                                                                                                                                                                                                         
          Check: resource.ComposeTestCheckFunc(                                                                                                                                                                                                                                                       
            testAccCheckAWSGlueJobExists(resourceName, &job),                                                                                                                                                                                                                                         
            resource.TestCheckResourceAttr(resourceName, "max_capacity", "0.0625"),                                                                                                                                                                                                                   
            resource.TestCheckResourceAttr(resourceName, "command.#", "1"),                                                                                                                                                                                                                           
            resource.TestCheckResourceAttr(resourceName, "command.0.script_location", "testscriptlocation"),                                                                                                                                                                                          
            resource.TestCheckResourceAttr(resourceName, "command.0.python_version", "2"),                                                                                                                                                                                                            
            resource.TestCheckResourceAttr(resourceName, "command.0.name", "pythonshell"),                                                                                                                                                                                                            
          ),                                                                                                                                                                                                                                                                                          
        },                                                                                                                                                                                                                                                                                            
        {                                                                                                                                                                                                                                                                                             
          ResourceName:      resourceName,                                                                                                                                                                                                                                                            
          ImportState:       true,                                                                                                                                                                                                                                                                    
          ImportStateVerify: true,                                                                                                                                                                                                                                                                    
        },                                                                                                                                                                                                                                                                                            
        {                                                                                                                                                                                                                                                                                             
          Config: testAccAWSGlueJobConfig_PythonShellWithVersion(rName, "3"),                                                                                                                                                                                                                         
          Check: resource.ComposeTestCheckFunc(                                                                                                                                                                                                                                                       
            testAccCheckAWSGlueJobExists(resourceName, &job),                                                                                                                                                                                                                                         
            resource.TestCheckResourceAttr(resourceName, "max_capacity", "0.0625"),                                                                                                                                                                                                                   
            resource.TestCheckResourceAttr(resourceName, "command.#", "1"),                                                                                                                                                                                                                           
            resource.TestCheckResourceAttr(resourceName, "command.0.script_location", "testscriptlocation"),                                                                                                                                                                                          
            resource.TestCheckResourceAttr(resourceName, "command.0.python_version", "3"),                                                                                                                                                                                                            
            resource.TestCheckResourceAttr(resourceName, "command.0.name", "pythonshell"),                                                                                                                                                                                                            
          ),                                                                                                                                                                                                                                                                                          
        },                                                                                                                                                                                                                                                                                            
        {                                                                                                                                                                                                                                                                                             
          ResourceName:      resourceName,                                                                                                                                                                                                                                                            
          ImportState:       true,                                                                                                                                                                                                                                                                    
          ImportStateVerify: true,                                                                                                                                                                                                                                                                    
        },                                                                                                                                                                                                                                                                                            
      },                                                                                                                                                                                                                                                                                              
    })                                                                                                                                                                                                                                                                                                
  }```